gokart/target.py

```diff
@@ -149,7 +149,8 @@ def _dump(self, obj) -> None:
         self._remove_temporary_directory()
 
     def _remove(self) -> None:
-        self._zip_client.remove()
+        if self._exists():
```

Comment:
@mski-iksm [ASK] Is it OK not to warn or raise Exception? This looks like not intended action to remove not existing  files.

Reply:
@Hi-king I think raising Exception is not good idea because this remove method is always called when TaskOnKart.rerun parameter is True.
https://github.com/m3dev/gokart/blob/master/gokart/task.py#L108

If we are to raise exception, we need to add existence check at TaskOnKart before calling target.remove().
But I think there is no good motivation of changing the attitude of TargetOnKart and TaskOnKart.
How do you think of this?

Reply:
@mski-iksm 

* Existence check on rerun seems to be good idea for me because it will be more clear what `rerun` intended to.
* As the same manner as standard library, Introduce flag `remove(missing_ok=True)` can be a solution
   * Pathlib has that interface https://docs.python.org/ja/3/library/pathlib.html#pathlib.Path.unlink
   * os.unlink raises FileNotFoundError
* It's OK to go with current solution :)

Reply:
@mski-iksm OK. Thank you for clarifying the point!